### PR TITLE
21 add api endpoints for crud for ngos

### PR DESCRIPTION
**Pull request by Miladepour**
## Description

**Adding endpoint for NGOs

## Related to

#21 

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: server/api.js
Comment (KptCnyt) [inline server/api.js:1]: Nice :)

**Review by KptCnyt — COMMENTED** [review]
Due to we have not linked the ngo list end points with current result page I am not able to check whether this one gets the datas from my database but validations and routing endpoints looks okay

**Review by KptCnyt — APPROVED** [review]

**Comment by Miladepour** [comment]
@EneVladMihai This pull request is ready for review.

**Review by natyfromwonderland — APPROVED** [review]
Thank you for adding the end points!

**Pull request merged**